tests/core/test_state_sync.py

```diff
@@ -1545,3 +1554,89 @@ def test_snapshot_batching(state_sync, mocker, make_snapshot):
     assert len(snapshots) == 3
     calls = mock.fetchall.call_args_list
     assert len(calls) == 2
+
+
+def test_migrate_only_to_add_sqlmesh_version_column(duck_conn, monkeypatch) -> None:
+    from sqlmesh import __version__ as SQLMESH_VERSION
+
+    n_migrations_before_sqlmesh_version = 31
```

Comment:
make this n -1 so we always test the latest migration right

Reply:
It's not that simple because this test has logic specific to the 31->32 migration, i.e. patching _update_versions and assuming that prior to that migration we don't have `sqlmesh_version`